### PR TITLE
refactor: retire Int part 7

### PR DESCRIPTION
**Pull request by ninaandrup**
#2588

### CODE REVIEW

File: main/src/ca/uwaterloo/flix/language/phase/Resolver.scala [hunk 1]
```diff
@@ -1475,7 +1475,7 @@ object Resolver {
       case "Char" => Type.mkChar(loc).toSuccess
       case "Float32" => Type.mkFloat32(loc).toSuccess
       case "Float64" => Type.mkFloat64(loc).toSuccess
-      case "Int" => Type.mkInt32(loc).toSuccess
+      //case "Int" => Type.mkInt32(loc).toSuccess
```
Comment (ninaandrup) [inline main/src/ca/uwaterloo/flix/language/phase/Resolver.scala:1478]: Should I delete this line?

Comment (ninaandrup) [reply main/src/ca/uwaterloo/flix/language/phase/Resolver.scala:1478]: All the tests pass now without it

Comment (magnus-madsen) [reply main/src/ca/uwaterloo/flix/language/phase/Resolver.scala:1478]: Yes :D Congratulations!

**Comment by magnus-madsen** [comment]
Woohoo!

**Comment by magnus-madsen** [comment]
@stephentetley FYI: `Int` has been retired as an alias for `Int32`. 

If this causes lots of breakages you can of course introduce the alias your self, but I would recommend against it long term.

**Comment by stephentetley** [comment]
Thanks for the heads up Magnus. I've mostly been changing `Int` to `Int32` in my code already. I think some slipped through in the tests of one of PR's I submitted over Christmas, though. But that was picked up in one of the earlier "retire-int" rounds.

**Pull request merged**